api/sample.env

```diff
@@ -412,6 +412,36 @@ AUTH_SECRET=Change me!
 # sample: SCO_ACCOUNT_RECOVERY_KEY_LIFETIME_MINUTES=10
 # SCO_ACCOUNT_RECOVERY_KEY_LIFETIME_MINUTES=
 
+
+# POLE EMPLOI authentication - Client ID
+# Refer to https://pole-emploi.io/data/api/pole-emploi-connect
+#
+# required for POLE EMPLOI authentication
```

Comment:
Il ne manque pas des lignes `# presence: optional` du coup  ?

Reply:
je vais mettre "required for POLE EMPLOI authentication, optional otherwise"

Reply:
Corrigé